src/WebJobs.Extensions.DurableTask.Scale/DurableTaskScaleExtension.cs

```diff
@@ -0,0 +1,75 @@
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Scale
+{
+    public class DurableTaskScaleExtension
+    {
+        private readonly IDurabilityProviderFactory durabilityProviderFactory;
+        private readonly DurabilityProvider defaultDurabilityProvider;
+        private readonly DurableTaskOptions options;
+		private readonly ILogger logger;
+        private readonly IEnumerable<IDurabilityProviderFactory> durabilityProviderFactories;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DurableTaskScaleExtension"/> class.
+        /// </summary>
+        /// <param name="options"></param>
+        /// <param name="logger"></param>
+        /// <param name="durabilityProviderFactories"></param>
+        public DurableTaskScaleExtension(
```

Comment:
I am hoping to make a new extension here, so that SC will load this one instead of DurableTaskExtension.

Reply:
That sounds like the right approach, but I also suggest we reduce the number of services that this new extension depends on.